packages/gatsby-theme-carbon/src/components/Aside/Aside.js

```diff
@@ -15,6 +15,8 @@ const Aside = ({ children, className, hideRule, ...rest }) => {
   );
 };
 
+export default Aside;
+
```

Comment:
why is this here if it's already being exported below?

Reply:
blergh.. at one point they were broken so i fixed them... then someone else fixed them on a different PR and merged to main.. then i merged from main... now i have two! mergemergemerge! :P